apis/v1alpha1/common.go

```diff
@@ -0,0 +1,15 @@
+package v1alpha1
+
+// Profile is common required information to extend provider-alibaba
+type Profile struct {
```

Comment:
Need to confirm all fields which are going to be used for extending OSS controller.